crates/re_space_view_spatial/src/visualizers/entity_iterator.rs

```diff
@@ -110,3 +109,125 @@ where
 
     Ok(())
 }
+
+// ---
+
+use re_query_cache::external::{paste::paste, seq_macro::seq};
+
+macro_rules! impl_process_archetype {
+    (for N=$N:expr, M=$M:expr => povs=[$($pov:ident)+] comps=[$($comp:ident)*]) => { paste! {
+        #[doc = "Cached implementation of [`process_archetype_views] for `" $N "` point-of-view components"]
```

Comment:
it took me a bit to understand what you mean with a pov-component, haven't seen that term before. Is that a thing now or do we have a better name for it?